server/src/main/java/io/crate/expression/predicate/NotPredicate.java

```diff
@@ -126,6 +127,7 @@ private static class SymbolToNotNullRangeQueryArgs extends SymbolVisitor<SymbolT
             Set.of(
                 AnyEqOperator.NAME,
                 AnyNeqOperator.NAME,
+                AndOperator.NAME,
```

Comment:
Seeing how there's already another issue https://github.com/crate/crate/issues/15083, I'm thinking maybe we should flip this around and instead list functions which don't require strict 3vl - I wonder which these even are.

I think we can make sure that operations like `NOT x` are fast by combining it with fields exist, but I'm not sure if anything more complicated than that would work out?

Reply:
👍